libseccomp-sys/build.rs

```diff
@@ -5,26 +5,239 @@
 
 use std::env;
 
+mod versions;
+
 const LIBSECCOMP_LIB_PATH: &str = "LIBSECCOMP_LIB_PATH";
 const LIBSECCOMP_LINK_TYPE: &str = "LIBSECCOMP_LINK_TYPE";
+const LIBSECCOMP_SRC_PATH: &str = "LIBSECCOMP_SRC_PATH";
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    println!("cargo:rerun-if-env-changed={}", LIBSECCOMP_SRC_PATH);
     println!("cargo:rerun-if-env-changed={}", LIBSECCOMP_LIB_PATH);
     println!("cargo:rerun-if-env-changed={}", LIBSECCOMP_LINK_TYPE);
 
-    if let Ok(path) = env::var(LIBSECCOMP_LIB_PATH) {
-        println!("cargo:rustc-link-search=native={}", path);
+    // We do cfg on blocks here instead of `if cfg!()...` so we don't have to build the cc library
+    // at all in the non-bundled case.
+
+    // If bundled feature is enabled, build the library and return any errors.
+    #[cfg(feature = "bundled")]
+    {
+        let lib_dir = build_bundled::build()?;
+        build_linked::link(Some(&lib_dir), Some("static"))
```

Comment:
Cloud you leave a comment that static linking is done by default in the bundled feature?